README.md

```diff
@@ -6,4 +6,39 @@
 [![Build Status](https://travis-ci.org/fluture-js/concurrify.svg?branch=master)](https://travis-ci.org/fluture-js/concurrify)
 [![Code Coverage](https://codecov.io/gh/fluture-js/concurrify/branch/master/graph/badge.svg)](https://codecov.io/gh/fluture-js/concurrify)
 
-Does nothing yet.
+Turn non-concurrent [FantasyLand 3][FL3] Applicatives concurrent.
+
+## Usage
+
+```js
+//The concurrify takes four arguments, explained below.
+const concurrify = require('concurrify');
+
+//We load the Type Representative of the Applicative we want to transform.
+const Future = require('fluture');
+
+//We create a "zero" instance and a "race" function for "Alternative".
+const zero = Future(() => {});
+const alt = Future.race;
+
+//We create an override "ap" function that works in parallel.
+const ap = (mx, mf) => Future.both(mx, mf).map(([x, f]) => f(x));
+
+//Calling concurrify with these arguments gives is a new Type Representative.
+const ConcurrentFuture = concurrify(Future, zero, alt, ap);
+
+//We can use our type as such:
+ConcurrentFuture(Future.of(1)).sequential.fork(console.error, console.log);
+```
+
+## API
+
+```hs
+concurrify :: (Applicative m, Alternative n)
+           => (TypeRep m, m a, (m a, m a) -> m a, (m a, m (a -> b)) -> m b)
+           -> n m
```

Comment:
I agree, the function approach looks most informative:

```hs
... -> m b -> n (m b)
```

But the above signature makes it look like `m b` is the inner type of our `n`, whilst actually when we map over it we get only `b` (and not `m b`). Since `concurrify` functions a little bit like a Monad Transformer, maybe we can take inspiration from how they do their type signatures? It's probably something like `n m a`.